plugins/proxies/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ set (proxies_out_dir "plugins")
 install ( FILES ${CMAKE_CURRENT_SOURCE_DIR}/lib_saltlib.ecllib DESTINATION ${proxies_out_dir} COMPONENT Runtime)
 install ( FILES ${CMAKE_CURRENT_SOURCE_DIR}/lib_keylib.ecllib DESTINATION  ${proxies_out_dir} COMPONENT Runtime)
 install ( FILES ${CMAKE_CURRENT_SOURCE_DIR}/lib_metaphone.ecllib DESTINATION ${proxies_out_dir} COMPONENT Runtime)
+install ( FILES ${CMAKE_CURRENT_SOURCE_DIR}/lib_metaphone3.ecllib DESTINATION ${proxies_out_dir} COMPONENT Runtime)
```

Comment:
Probably also need to remove the corresponding line from the LN repo

Reply:
Where is it?  I searched and failed to find it.

Reply:
/Users/rchapman/LN/plugins/metaphone3/CMakeLists.txt:install ( TARGETS metaphone3 DESTINATION plugins )

Reply:
(I don't know what version of LN I currently have checked out...)

Reply:
Ok, that's the so. I share your bafflement